README.md

```diff
@@ -220,14 +220,15 @@ Install the [Multi-User Kubeflow Pipelines](https://www.kubeflow.org/docs/compon
 ```sh
 kustomize build apps/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user | kubectl apply -f -
 ```
+This installs argo with the safe-to use runasnonroot emissary executor. You still have to make sure yourself that your pipeline main containers
+are runasnonroot, but at least this fixes it for Argo. Please be aware of the root security risks.
```

Comment:
Perhaps rewording (although we could cut sentence #2)....Executing this statement installs Argo with a safe-to-use runasnonroot emissary executor.    Please note that the installer is still responsible  to analyze the security issues opened when containers are run with root access and to decide if the kubeflow pipeline main containers are run as runasnonroot.  It is strongly recommended that the pipelines main containers are installed and run as runasnonroot to mitigate security risks.